treeherder/seta/high_value_jobs.py

```diff
@@ -0,0 +1,148 @@
+import logging
+
+from treeherder.etl.seta import Treecodes
+
+LOG = logging.getLogger(__name__)
+
+
+def get_distinct_tuples(repo_name='mozilla-inbound'):
+    '''
+    Returned data:
+      [("windows8-64", "debug", "web-platform-tests-1"), ...]
+    '''
+    return Treecodes(repo_name).query_jobtypes()
+
+
+def is_matched(failure, removals):
+    found = False
+    if failure in removals:
+        found = True
+    else:
+        # XXX: Why do we also compare three other ways ''?
+        #      See if we can get rid of this
```

Comment:
I found we don't need the blank values- also for optimizations we should pull changes done a couple weeks ago to pack the unique job tuple into a 4byte int and use that for comparison.  Just removing the need for comparing '' will be a nice win.